lib/ui/window/platform_configuration.h

```diff
@@ -413,9 +413,10 @@ class PlatformConfiguration final {
   ///
   /// @param[in] window_id The id of the window to find and return.
   ///
-  /// @return     a pointer to the Window.
+  /// @return     a pointer to the Window. Nullptr if the ID is not registered
+  ///             yet.
```

Comment:
Do you remember what our plan is for `Canvas::drawShadow`? It uses the implicit view's device pixel ratio:

https://github.com/flutter/engine/blob/1fa222fae28354a3e9e1d0b73190b0a4b2564731/lib/ui/painting/canvas.cc#L624-L628

If the implicit view is disabled, this will crash

Reply:
Oh right, I totally forgot. I'll open an issue to discuss this.

Reply:
https://github.com/flutter/flutter/issues/130165